data/json/items/fluff.json

```
@@ -239,6 +239,278 @@
+    "weight": "300 g",
+    "volume": "1800 ml",
+    "price": 25,
+    "price_postapoc": 100,
+    "material": [ "cardboard" ],
+    "symbol": "?",
+    "color": "green",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "ludo",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Ludo" },
+    "description": "Ludo is a strategy board game for two to four players, in which the players race their four tokens from start to finish according to the rolls of a single dice.",
+    "weight": "420 g",
+    "volume": "2981 ml",
+    "price": 25,
+    "price_postapoc": 100,
+    "material": [ "cardboard" ],
+    "symbol": "?",
+    "color": "white",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "poppirate",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str": "Pop-up Pirate" },
+    "description": "It was once a popular luck-based game.  The pirate is placed into a spring-loaded barrel and rotated to randomize the unlucky slot.  Players must take it in turns to insert plastic swords into slots in the side of the barrel.  If a player inserts the sword into a specific slot (which changes randomly every time the game is played), the pirate is launched out of the barrel and the player is eliminated.  The last player remaining after all others have been eliminated wins.",
+    "weight": "500 g",
+    "volume": "8519 L",
+    "price": 25,
+    "price_postapoc": 100,
+    "material": [ "plastic" ],
+    "symbol": "?",
+    "color": "brown",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "ambush",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str_sp": "Ambuscade!" },
+    "description": "A fun man-to-man board wargame.  It can be played completely alone or with friends, where when playing alone, the game scenario itself controls the opponent, via tables, charts and a 'paragraph book' which the player refer to see how the opponent is reacting.",
+    "weight": "1020 g",
+    "volume": "3363 ml",
+    "price": 25,
+    "price_postapoc": 100,
+    "material": [ "cardboard" ],
+    "symbol": "?",
+    "color": "green",
+    "flags": [ "NO_REPAIR" ],
+    "use_action": [ "PLAY_GAME" ]
+  },
+  {
+    "id": "ranger",
+    "type": "GENERIC",
+    "category": "other",
+    "name": { "str_sp": "Patrol Leader" },
+    "description": "Patrol Leader is a tactical solitaire board game.  Is this game, the player is put in the role of a patrol leader in different modern battlefield scenarios, performing missions, capturing or killing enemies, protecting its teammates and advancing in the map. Very accurate and realistic, guaranteed fun if you like board games.",
```

```suggestion
    "description": "Patrol Leader is a tactical solitaire board game.  Is this game, the player is put in the role of a patrol leader in different modern battlefield scenarios, performing missions, capturing or killing enemies, protecting its teammates and advancing in the map.  Very accurate and realistic, guaranteed fun if you like board games.",
```
Final spacing error to fix.